src/lib/token.js

```diff
@@ -0,0 +1,15 @@
+import jwt from "jsonwebtoken";
+
+export const generateToken = (data, secret, expiresInSeconds) => jwt.sign(
+    { ...data },
+    secret,
+    { expiresIn: `${expiresInSeconds} seconds`, algorithm: "HS256" }
```

Comment:
If `expiresInSeconds` is not a number (for example an empty string, or a string with spaces, text, etc) the underlying time parsing library returns undefined (I tested it myself). What is the impact here? Does the function crash? Or is there any sort of potential bypass?

Reply:
The `expiresIn` doesnt use parameters coming from the end user, so there is no problem.
I will however evaluate the result of the case you mentioned

Reply:
If I send a string to the sign function it throws an error.